### ISSUE TITLE
Bluetooth in 20240614 is very buggy

### ISSUE THREAD

**Issue by oppressor1761** [GrapheneOS/os_issue_tracker#3666]
P8P 20240614.
Sometimes when open Bluetooth from Quick Settings, my phone automatically locked screen as if I touched power button (which I'm definitely not). No error log
Frequency: 30% these days
After I unlock screen, I can turn on Bluetooth normally. Basically no functionality affected. 

Another glitch: 
Sometimes when open Bluetooth from Quick Settings, Bluetooth automatically turns off immediately and turns back on again after 5 seconds or so. No logs 
Frequency: 30%
Basically no functionality affected
